clouddq/templates/dbt/dbt_project.yml

```diff
@@ -42,9 +42,13 @@ clean-targets: # directories to be removed by `dbt clean`
 models:
   data_quality:
     data_quality_engine:
-      materialized: table
+      +materialized: ephemeral
```

Comment:
I'm a bit confused by the "ephemeral" type here. This model relates to the dq_summary table which is materialised as a table (not expiring), right? Would the ephemeral translate to the TEMPORAL table in BQ, as opposed to materialised table with expiration?

Reply:
ephemeral is effectively a CTE. This config here defines the default table materialization config, which can be overridden in the model file itself. Effectively, we have two models in data_quality_engine: main.sql which is a CTE/ephemeral materialization, and dq_summary.sql which is an incremental materialization.

Reply:
ok, thanks!